src/app/components/menu/menu.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { MenuComponent } from 'src/app/components/menu/menu.component';
+import { CreateButtonModule } from 'src/app/components/create-button/create-button.module';
+
+@NgModule({
+  declarations: [MenuComponent],
+  imports: [CommonModule, CreateButtonModule],
+  providers: [],
+  exports: [MenuComponent],
+})
+export class MenuModule {}
```

Comment:
```suggestion
})

export class MenuModule {}
```

Reply:
decorator and class shouldn't have an empty line in between.[ guideline](https://angular.io/guide/styleguide)